jib-cli/src/main/java/com/google/cloud/tools/jib/cli/cli2/JibCli.java

```diff
@@ -0,0 +1,357 @@
+import java.util.Optional;
+import picocli.CommandLine;
+import picocli.CommandLine.ArgGroup;
+import picocli.CommandLine.Option;
+
+@CommandLine.Command(
+    name = "jib",
+    versionProvider = VersionInfo.class,
+    mixinStandardHelpOptions = true,
+    showAtFileInUsageHelp = true,
+    synopsisSubcommandLabel = "COMMAND",
+    description = "A tool for creating container images")
+public class JibCli {
+  @Option(
+      names = "--verbosity",
+      paramLabel = "<level>",
+      defaultValue = "lifecycle",
+      description = "set logging verbosity (error, warn, lifecycle (default), info, debug)")
+  @SuppressWarnings("NullAway.Init") // initialized by picocli
+  private String verbosity;
+
+  // Hidden debug parameters
+  @Option(names = "--stacktrace", hidden = true)
+  @SuppressWarnings("NullAway.Init") // initialized by picocli
+  private boolean stacktrace;
+
+  @Option(names = "--http-trace", hidden = true)
+  @SuppressWarnings("NullAway.Init") // initialized by picocli
+  private boolean httpTrace;
+
+  @Option(names = "--serialize", hidden = true)
+  @SuppressWarnings("NullAway.Init") // initialized by picocli
+  private boolean serialize;
+
+  // Build Configuration
+  @Option(
+      names = {"-t", "--target"},
+      required = true,
+      paramLabel = "<target-image>",
+      description =
+          "The destination image reference or jib style url,%nexamples:%n gcr.io/project/image,%n registry://image-ref,%n docker://image,%n tar://path")
+  @SuppressWarnings("NullAway.Init") // initialized by picocli
+  private String targetImage;
+
+  @Option(
+      names = {"-c", "--context"},
+      defaultValue = ".",
+      paramLabel = "<project-root>",
+      description = "The context root directory of the build (ex: path/to/my/build/things)")
+  @SuppressWarnings("NullAway.Init") // initialized by picocli
+  private Path contextRoot;
+
+  @Option(
+      names = {"-b", "--build-file"},
+      paramLabel = "<build-file>",
+      description = "The path to the build file (ex: path/to/other-jib.yaml)")
+  @SuppressWarnings("NullAway.Init") // initialized by picocli
+  private Path buildFile;
+
+  // unfortunately we cannot verify for --target=tar://... this is required, we must do this after
+  // pico cli is done parsing
+  @Option(
+      names = "--name",
+      paramLabel = "<image-reference>",
+      description =
+          "The image reference to inject into the tar configuration (required when using --target tar://...)")
+  @SuppressWarnings("NullAway.Init") // initialized by picocli
+  private String name;
+
+  @Option(
+      names = {"-p", "--parameter"},
+      paramLabel = "<name>=<value>",
+      description =
+          "templating parameter to inject into build file, replace $${<name>} with <value> (repeatable)")
+  @SuppressWarnings("NullAway.Init") // initialized by picocli
+  private Map<String, String> templateParameters = new HashMap<String, String>();
+
+  @Option(
+      names = "--tags",
```

Comment:
How about calling this `--additional-tags`?

I've seen many occasions people put `latest` into `<tags>` unnecessarily. And actually our code can cause confusion too, as we call this `getTags()` at the front (as the param name is `<tags>`) and later `getAdditionalTags()`. `BuildContext` also has `getAllTargetImageTags()` and `setAdditionalTargetImageTags()` to distinguish them.